anchore_engine/clients/services/internal.py

```diff
@@ -284,7 +284,13 @@ def dispatch(
         else:
             filtered_qry_params = None
 
-        log_body = ensure_str(body[:512]) + "..." if body and len(body) > 512 else body
+        try:
+            log_body = (
+                ensure_str(body[:512]) + "..." if body and len(body) > 512 else body
+            )
+        except UnicodeError as e:
```

Comment:
This is catching and assigning the exception to `e` but not doing anything with it

Reply:
resolved by @zburstein